general/server/storage.md

```diff
@@ -0,0 +1,30 @@
+---
+uid: server-storage
+title: Storage
+---
+
+## Storage
+
+Jellyfin is designed to directly read media from the filesystem. This means to pass a network storage device that is using samba or NFS must be directly mounted to the OS. The Jellyfin database also should be stored locally and not on a network storage device.
+
+## Docker/VM's
+
+For storage, a library database can take up to 50 to 100 GB. The [transcoding](https://github.com/jellyfin/jellyfin-docs/blob/master/general/server/transcoding.md) folder needs roughly the same size as the original media if it's being transcoded at the same bitrate. A single 50GB Blu-Ray Remux by itself can take up to approximately 60GB or as little as 15GB, depending on the quality selected. If the transcoding folder is held on the same storage as the database, this must be taken into consideration.
+
+## Cloud
+
+A popular choice for cloud storage has been the program [rclone](https://rclone.org/downloads/). It is supported on most Operating Systems. To prevent API bans, rclone is typically paired with another program such as [mergerfs](https://github.com/trapexit/mergerfs). For cloud storage, it is recommended to disable image extraction as this requires downloading the entire file to perform this task. Unfortunately, the image extractor can't be [turned off](https://github.com/jellyfin/jellyfin/issues/2355) in Jellyfin at the moment causing [performance issues](https://github.com/jellyfin/jellyfin/issues/2600).
```

Comment:
You just said it's recommended to turn image extraction off, and then say that it can't be turned off.  I realize it's a bug, but it's rather confusing at the moment.

Reply:
I turned it into a note